pkg/minikube/cluster/mount.go

```diff
@@ -54,19 +54,38 @@ type mountRunner interface {
 	RunCmd(*exec.Cmd) (*command.RunResult, error)
 }
 
+const (
+	// MountErrorUnknown failed with unknown error
+	MountErrorUnknown = iota
+	// MountErrorConnect
+	MountErrorConnect
+)
+
+type MountError struct {
```

Comment:
add comment for all exported types. (or dont make it exported if it hasn't been called outside this package

Reply:
Done. It needs to be exported, else we can't check the ErrorType.